templates/base/head_script.tmpl

```diff
@@ -39,6 +39,7 @@ If you introduce mistakes in it, Gitea JavaScript code wouldn't run correctly.
 			error_occurred: '{{.locale.Tr "error.occurred"}}',
 			network_error: '{{.locale.Tr "error.network_error"}}',
 			remove_label_str: '{{.locale.Tr "remove_label_str"}}',
+			rename_branch_to: '{{.locale.Tr "repo.branch.rename_branch_to"}}',
```

Comment:
It doesn't seem to be a "general" string

Reply:
Yes, I noticed the comment, but it didn't occur to me that I could add a `<script>` in `list.tmpl` at that time 😂. Fix it now